Makefile

```diff
@@ -1,34 +1,48 @@
 EXTRA_ARGS :=
 
+ifneq ($(OS),Windows_NT)
 UNAME := $(shell uname)
 ifeq ($(UNAME), Linux)
 NPROC = $(shell nproc)
 endif
 ifeq ($(UNAME), Darwin)
 NPROC = $(shell sysctl -n hw.physicalcpu)
 endif
+endif
 
 #########
 # BUILD #
 #########
 .PHONY: requirements develop build build-debug build-conda install
 
 requirements:  ## install python dev and runtime dependencies
+ifeq ($(OS),Windows_NT)
+	Powershell.exe -executionpolicy bypass -noprofile ./make_requirements.ps1
+else
 	python -m pip install toml
 	python -m pip install `python -c 'import toml; c = toml.load("pyproject.toml"); print("\n".join(c["build-system"]["requires"]))'`
 	python -m pip install `python -c 'import toml; c = toml.load("pyproject.toml"); print("\n".join(c["project"]["optional-dependencies"]["develop"]))'`
+endif
 
 develop: requirements  ## install dependencies and build library
 	python -m pip install -e .[develop]
 
 build:  ## build the library
+ifeq ($(OS),Windows_NT)
+	python setup.py build build_ext --inplace
```

Comment:
the `setup.py` / `CMakeLists.txt` can set the default parallelism, so we can simplify by just doing `--inplace` (we don't need the `-- -- -j$(NPROC)`